server/taskflows/hpccloud/taskflow/paraview/pvw.sh

```diff
@@ -47,7 +47,7 @@ export DISPLAY
 
 # First run pvpython
 ${PV_PYTHON} {{'--mesa-llvm' if not gpu else ''}} ${VISUALIZER} --timeout 3600 --host $IPADDRESS --port ${WEBSOCKET_PORT} \
-{{ '--data-dir %s' % dataDir if dataDir else ''}} \
+{{ '--data %s' % dataDir if dataDir else ''}} \
 {{ '--load-file %s' % fileName if fileName else '' }}
```

Comment:
That change is in Visualizer (which bring the python file) and since the new version is pulled in, I needed to fix it.

--data-dir was replace by --data to match all the other paraviewweb apps (ArcticViewer, LightViz) command line arguments. Specially since the Visualizer command line was using --data.